tools/backups2datalad/util.py

```diff
@@ -252,7 +252,7 @@ def get_deleted(self, config: Config) -> Iterator[str]:
         """
         for apath in self.local_assets:
             if config.match_asset(apath):
-                del self.asset_metadata[apath]
+                self.asset_metadata.pop(apath, None)
```

Comment:
ah, you mean to not fail if `apath` isn't there in the dict? gotcha now.

Reply:
I'm not entirely sure what would cover it.  As far as I can tell, the only way `del` should have failed would be if an asset was absent from `assets.json` but present in the dataset, and I don't think that should happen.